wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,444 @@
+# Usage permissions
+
+osu! is centred around being able to freely make and share creations with one another. It is important ensure that appropriate permissions have been acquired to use any other creator's work in new creations, and that all such work is properly attributed.
+
+## Opt-in
+
+For artists wishing to add or update their entry on this listing, please feel free to reach out to the osu! music licensing team at [music@ppy.sh](mailto:music@ppy.sh).
+
+## Audio
+
+*Notice: Posting or streaming gameplay on other sites may potentially be subject to different licensing requirements that need to be negotiated or understood. Please exercise due diligence when broadcasting gameplay to an audience.*
+
+This section details which artists are willing to allow use of their audio works in osu!, and under what conditions (where appropriate).
+
+All tracks listed on any [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) are approved for use in the creation of osu! beatmaps. Tracks from a Featured Artist that do not appear in their listing are *not* licensed, and additional permissions may be required to use them.
+
+### Allowed
+
+| Artist |  | Status |
+| :-- | :-- | :-- |
+| [![][FA]](https://osu.ppy.sh/beatmaps/artists/18) | [\*namirin](https://osu.ppy.sh/beatmaps/artists/18) | ![][true] |
```

Comment:
would make more sense to have the pill link to the [FA wiki article](https://osu.ppy.sh/wiki/en/People/Featured_Artists) instead of linking to the same artist page twice in a row.

Reply:
there's less utility in almost 400 links to the fa article than links to each individual artist. between having or not having a clickable icon, this is the best option imo